### DISCUSSION TITLE
Pypi maintener

### DISCUSSION THREAD

**Discussion by hermann74** [scikit-mine/scikit-mine#143] [General]
Bonjour @remiadon ,
 @thomasbtnfr  et moi aimerions mettre à jour l'email du paquet Pypi, skm-dev@inria.fr
Peux tu transférer les droits sur le nouveau compte Pypi : username = skmineTeam
```
Log in, go to the “Collaborators” section in the settings 
(https://pypi.org/manage/project/<your-project>/collaboration/),
 add the new maintainer as Owner.
The new maintainer can then remove you from the project.
```
Merci de ta collaboration

**Comment by remiadon** [comment]
Bonjour @hermann74 
Je viens d'ajouter skmineTeam sur le compte Pypi. N'hésite pas à me dire si un problème persiste.

Merci de faire vivre ce projet
